From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Fri, 18 Aug 2023 00:12:23 +0200
Subject: layout: Export PressureBarrier

It's an utility function that extensions can use to create pointer barriers,
so export it to allow wider usage

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2900
---
 js/ui/layout.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/layout.js b/js/ui/layout.js
index 5611dad..d1053df 100644
--- a/js/ui/layout.js
+++ b/js/ui/layout.js
@@ -1299,7 +1299,7 @@ class HotCorner extends Clutter.Actor {
     }
 });
 
-class PressureBarrier extends Signals.EventEmitter {
+export class PressureBarrier extends Signals.EventEmitter {
     constructor(threshold, timeout, actionMode) {
         super();
 
